.editorconfig

```diff
@@ -0,0 +1,16 @@
+root = true
+[*]
+indent_style=tab
+indent_size=tab
+tab_width=4
+end_of_line=lf
+charset=utf-8
+trim_trailing_whitespace=true
+max_line_length=100
+insert_final_newline=true
+
+[*.yml]
```

Comment:
We don't have yamls in here :D

Reply:
I shamefully copied the file from substrate :grin: